lib/webdrivers/chromedriver.rb

```diff
@@ -43,6 +47,42 @@ def downloads
         end
       end
 
+      # Returns major version from the currently installed Chrome version
+      def chrome_major_version
+        chrome_version[0..1]
```

Comment:
You might want to grab everything up to the first dot instead of just the first 2 characters. Otherwise, you'll have to update this in a year or two when version 100 is released.

Reply:
Good point! The PR has been updated.